app/controllers/medical_records_controller.rb

```diff
@@ -65,6 +65,6 @@ def set_medical_record
 
     # Only allow a list of trusted parameters through.
     def medical_record_params
-      params.require(:medical_record).permit(:patient_id, :patology, :habits, :current_state, :notes)
+      params.require(:medical_record).permit(:patient_id, :patology, :habits, :current_state, :notes, :reason_consultation, :toxicological_habits, :hereditary_family_history, :non_pathological_personal_record)
```

Comment:
`toxicological_habits` column was renamed.
```suggestion
      params.require(:medical_record).permit(:patient_id, :patology, :habits, :current_state, :notes, :reason_consultation, :hereditary_family_history, :non_pathological_personal_record)
```